acct-enricher-job/pom.xml

```diff
@@ -112,10 +113,21 @@
             <id>ossrh</id>
             <url>https://oss.sonatype.org/content/repositories/snapshots</url>
         </repository>
+        <repository>
+            <id>gitlab-maven</id>
+            <url>https://gitlab.com/api/v4/projects/37429227/packages/maven</url>
+        </repository>
     </repositories>
 
     <dependencies>
 
+        <!-- https://gitlab.com/ubirch/ubirch-cassandra-utils -->
+        <dependency>
+            <groupId>com.ubirch</groupId>
+            <artifactId>ubirch-cassandra-utils_${scala.compat.version}</artifactId>
+            <version>${ubirch-cassandra-utils.version}</version>
+        </dependency>
+
         <!-- https://mvnrepository.com/artifact/io.getquill/quill-cassandra-monix -->
         <dependency>
             <groupId>io.getquill</groupId>
```

Comment:
Should we not upgrade this lib too?

Reply:
I already updated it in the base branch(https://github.com/ubirch/ubirch-accounting-service/tree/feature/UC-624-update-cassandra).